scripts/build/get_source/termux_git_clone_src.sh

```diff
@@ -37,9 +71,11 @@ termux_git_clone_src() {
 
 		popd
 
-		echo "$TERMUX_PKG_VERSION" > $TMP_CHECKOUT_VERSION
+		echo "$TERMUX_PKG_VERSION" > "$TMP_CHECKOUT_VERSION"
+	else
+		echo "Skipped downloading of git source from '$termux_pkg_srcurl'"
 	fi
 
-	rm -rf $TERMUX_PKG_SRCDIR
-	cp -Rf $TMP_CHECKOUT $TERMUX_PKG_SRCDIR
+	rm -rf "$TERMUX_PKG_SRCDIR"
+	cp -Rf "$TMP_CHECKOUT" "$TERMUX_PKG_SRCDIR"
```

Comment:
Probably it is out of scope of this PR but we should add checksum validating here.
```
	local s=$(find . -type f ! -path '*/.git/*' -print0 | xargs -0 sha256sum | LC_ALL=C sort | sha256sum)
	if [[ "${s}" != "${TERMUX_PKG_SHA256}  "* ]]; then
		termux_error_exit "Checksum mismatch for source files.\nActual: ${s}\n Expected: ${TERMUX_PKG_SHA256}"
	fi
```